packages/api-gateway/src/drivers/http/adapters/index.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
 
-const mathAdapters = require('./math/handlers');
```

Comment:
@Alejandrocuartas we shouldn't remove the mathAdapters until all squads are their packages and perform their integrations.